app/services/application_state_change.rb

```diff
@@ -5,8 +5,8 @@ class ApplicationStateChange
   # as they do not need references or the 7-day cooling off period
   STATES_THAT_MAY_BE_SENT_TO_PROVIDER = %i[application_complete unsubmitted].freeze
   STATES_NOT_VISIBLE_TO_PROVIDER = %i[unsubmitted awaiting_references application_complete cancelled].freeze
-  STATES_VISIBLE_TO_PROVIDER = %i[awaiting_provider_decision offer pending_conditions recruited rejected declined withdrawn conditions_not_met offer_withdrawn rejected_at_end_of_cycle].freeze
-  ACCEPTED_STATES = %i[pending_conditions conditions_not_met recruited].freeze
+  STATES_VISIBLE_TO_PROVIDER = %i[awaiting_provider_decision offer pending_conditions recruited rejected declined withdrawn conditions_not_met offer_withdrawn rejected_at_end_of_cycle offer_deferred].freeze
+  ACCEPTED_STATES = %i[pending_conditions conditions_not_met recruited offer_deferred].freeze
```

Comment:
Should this go into `OFFERED_STATES` instead? cc @duncanjbrown as it impacts TAD reporting.

Reply:
In fact this _is_ an accepted state bc it's not possible to enter it without first accepting the offer. However I take your point that TAD might not want to count it. Propose leaving it in here and either defining another constant below (eg `ACCEPTED_STATES_FOR_TAD`) or doing some work in the TAD export to filter it out. Think I prefer the latter?

Reply:
Thanks, I will have a go at filtering it out of the TAD export.